var/spack/repos/builtin/packages/albert/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Albert(MakefilePackage):
+    """Albert is an interactive program to assist the
+    specialist in the study of nonassociative algebra."""
+
+    homepage = "https://people.cs.clemson.edu/~dpj/albertstuff/albert.html"
+    url      = "https://github.com/kentavv/Albert/archive/v4.0a_opt4.tar.gz"
+
+    version('4.0a_opt4', '79e3d9623602f2ca5db7d84c81d4eb8c')
+
+    depends_on('readline', type='build')
```

Comment:
Is `readline` really only a build dependency? Most packages that depend on `readline` link to it as well.